### PR TITLE
Feat(web-twig): Add ScrollView component #DS-465

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: packages/web-twig/src/Resources/components/ScrollView/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# ScrollView
+
+This is Twig implementation of the [ScrollView] component.
```
Comment (literat) [inline packages/web-twig/src/Resources/components/ScrollView/README.md:3]:
```suggestion
This is the Twig implementation of the [ScrollView] component.
```

File: packages/web-twig/src/Resources/components/ScrollView/ScrollView.twig [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _rootClassName, _rootDirectionClassName, _styleProps.className ] -%}
+{%- set _mainPropsWithoutReservedAttributes = props | filter((value, prop) => prop != 'data-direction') -%}
+
+<div
+    {{ mainProps(_mainPropsWithoutReservedAttributes) }}
+    {{ styleProp(_styleProps) }}
+    {{ classProp(_classNames) }}
+    data-direction="{{ _direction }}"
```
Comment (literat) [inline packages/web-twig/src/Resources/components/ScrollView/ScrollView.twig:21]:
```suggestion
    data-spirit-direction="{{ _direction }}"
```

**Review by literat — COMMENTED** [review]
It is looking good. Please use the `data-spirit` prefix in all cases expect `data-toggle` which is BC.

**Review by tomassychra — APPROVED** [review]
👍

**Review by literat — APPROVED** [review]
LGTM 👍

File: yarn.lock
Comment (adamkudrna) [inline yarn.lock:1]: Why a change in this file?

File: packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewHiddenScrollbar.twig [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<section class="docs-Section">
+
+    <h2 class="docs-Heading">Hidden Scrollbar</h2>
+
+    <div class="mb-1000" style="height: 160px">
+
+        <ScrollView
+            direction="vertical"
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewHiddenScrollbar.twig:8]: `direction` can be omitted, `vertical` is the default value.

File: packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingIndicators.twig [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            </p>
+        </ScrollView>
+
+    </div>
+
+    <ScrollView
+        data-toggle="scrollView"
+        direction="horizontal"
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingIndicators.twig:22]: Maybe `direction` first, like in a previous story?

File: packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingIndicators.twig [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
```
Comment (adamkudrna) [inline packages/web-twig/src/Resources/components/ScrollView/stories/ScrollViewScrollingIndicators.twig:48]: Again, the property order (up to you).

File: packages/web-twig/tests/components-fixtures/scrollViewDefault.twig
Comment (adamkudrna) [inline packages/web-twig/tests/components-fixtures/scrollViewDefault.twig:1]: Please separate the test cases with comments so we better know what is being tested.

See `FileUploaderInput` test for inspiration:

```html
<FileUploaderInput
    id="example-input"
/>

<!-- Render with custom texts -->
<!-- … -->

<!-- Render in danger state -->
<!-- … -->

<!-- Render with key HTML attributes transferred onto the native input -->
<!-- … -->

<!-- Render with UNSAFE props -->
<!-- … -->

<!-- Render with all props -->
<!-- … -->
```

**Review by adamkudrna — APPROVED** [review]
Cool! 👍🏻

File: yarn.lock
Comment (crishpeen) [reply yarn.lock:1]: It was a fix before #851, removed.

**Pull request merged**